### PR TITLE
Dynamic Multistore

### PR DESCRIPTION
**Pull request by VadymSachenko**
## PR Description
This PR updates the Dynamic Multistore feature document.
Jira: https://spryker.atlassian.net/browse/DOC-215
TBD

## Checklist
- [ ] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: docs/pbc/all/dynamic-multistore/202307.0/dynamic-multistore.md [hunk 1]
```diff
@@ -1,19 +1,84 @@
...
+
+![product-prices](https://spryker.s3.eu-central-1.amazonaws.com/docs/pbc/all/dynamic-multistore/dynamic-multistore.md/product-prices.png)
+
+## Viewing the store on the storefront
+
+The newly created store is viewable on the storefront, complete with the assigned products and prices.
+
+![store-switcher](https://spryker.s3.eu-central-1.amazonaws.com/docs/pbc/all/dynamic-multistore/dynamic-multistore.md/store-switcher.png)
+
+![search-suggestions](https://spryker.s3.eu-central-1.amazonaws.com/docs/pbc/all/dynamic-multistore/dynamic-multistore.md/search-suggestions.png)
```
Comment (VadymSachenko) [inline docs/pbc/all/dynamic-multistore/202307.0/dynamic-multistore.md:59]: Add a description to this screenshot

File: docs/pbc/all/dynamic-multistore/202307.0/dynamic-multistore.md [hunk 2]
```diff
@@ -1,19 +1,84 @@
...
+
+![assigned-product-searchable-in-the-storefront](https://spryker.s3.eu-central-1.amazonaws.com/docs/pbc/all/dynamic-multistore/dynamic-multistore.md/assigned-product-searchable-in-the-storefront.png)
```
Comment (VadymSachenko) [inline docs/pbc/all/dynamic-multistore/202307.0/dynamic-multistore.md:61]: Add a description to this screanshot

File: docs/pbc/all/dynamic-multistore/202307.0/dynamic-multistore.md [hunk 3]
```diff
@@ -1,19 +1,84 @@
...
+
+When the new queue infrastructure is published, the store becomes visible on the storefront.
+
+## Assigning products and their prices
+
+Products, along with their prices, can be assigned to a product using the Back Office UI or other methods of creating a new product in the system. This offers flexibility in managing the product catalog.
+
+![store-relation](https://spryker.s3.eu-central-1.amazonaws.com/docs/pbc/all/dynamic-multistore/dynamic-multistore.md/store-relation.png)
+
+![product-prices](https://spryker.s3.eu-central-1.amazonaws.com/docs/pbc/all/dynamic-multistore/dynamic-multistore.md/product-prices.png)
```
Comment (VadymSachenko) [inline docs/pbc/all/dynamic-multistore/202307.0/dynamic-multistore.md:51]: Add a description to this screanshot

File: docs/pbc/all/dynamic-multistore/202307.0/dynamic-multistore.md [hunk 4]
```diff
@@ -1,19 +1,84 @@
...
+The *Dynamic Multistore* feature lets you create and manage multiple online stores within the same region of the Back Office installation. It streamlines the setup and maintenance of distinct stores tailored to various customer segments, regions, or product categories. 
 
-Key benefits:
-* Intuitive interface: the Back Office interface is designed for ease of use, allowing even non-technical users to navigate and set up stores effortlessly.
-* Store customization: business users can customize each store's branding, theme, and appearance to maintain a unique identity that resonates with their target market.
-* Seamless provisioning: the application doesn't need to be redeployed to add or update stores.
+In the following sections, you can learn more details about the capabilities of Dynamic Multistore.
+## Viewing stores
+
+In the Back Office, in the **Administration&nbsp;<span aria-label="and then">></span>Stores**, you can view the list of stores in the current region. The **Stores** page offers a consolidated view of all the stores within a specific region.
```
Comment (VadymSachenko) [inline docs/pbc/all/dynamic-multistore/202307.0/dynamic-multistore.md:19]:
```suggestion
In the Back Office, in the **Administration&nbsp;<span aria-label="and then">></span> Stores**, you can view the list of stores in the current region. The **Stores** page offers a consolidated view of all the stores within a specific region.
```

File: _includes/pbc/all/install-features/202307.0/install-dynamic-multistore-the-cart-feature.md [hunk 5]
```diff
@@ -3,7 +3,8 @@
 Dynamic Multistore is part of an *Early Access Release*. This *Early Access* feature introduces the ability to handle the store entity in the Back Office. Business users can try creating stores without editing the `Stores.php` file and redeploying the system.
 
 {% endinfo_block %}
-This document describes how to install Dynamic Store + the Cart feature.
+
+This document describes how to install the [Dynamic Multistore](/docs/pbc/all/dynamic-multistore/{{page.version}}/base-shop/dynamic-multistore-feature-overview.html) + [Cart](/docs/pbc/all/cart-and-checkout/{{page.version}}/base-shop/cart-feature-overview/cart-feature-overview.html) feature.
```
Comment (andriitserkovnyi) [inline _includes/pbc/all/install-features/202307.0/install-dynamic-multistore-the-cart-feature.md:7]:
```suggestion
This document describes how to install [Dynamic Multistore](/docs/pbc/all/dynamic-multistore/{{page.version}}/base-shop/dynamic-multistore-feature-overview.html) + the [Cart](/docs/pbc/all/cart-and-checkout/{{page.version}}/base-shop/cart-feature-overview/cart-feature-overview.html) feature.
```

Comment (andriitserkovnyi) [reply _includes/pbc/all/install-features/202307.0/install-dynamic-multistore-the-cart-feature.md:7]: Please do the same in the rest of the docs

File: _includes/pbc/all/install-features/202307.0/install-dynamic-multistore.md [hunk 6]
```diff
@@ -2,10 +2,13 @@
 
 Dynamic Multistore is part of an *Early Access Release*. This *Early Access* feature introduces the ability to handle the store entity in the Back Office. Business users can try creating stores without editing the `Stores.php` file and redeploying the system.
 
-{% endinfo_block %} 
+{% endinfo_block %}
+
+This document describes how to integrate the [Dynamic Multistore](/docs/pbc/all/dynamic-multistore/{{page.version}}/base-shop/dynamic-multistore-feature-overview.html) feature into a Spryker project.
```
Comment (andriitserkovnyi) [inline _includes/pbc/all/install-features/202307.0/install-dynamic-multistore.md:7]:
```suggestion
This document describes how to install [Dynamic Multistore](/docs/pbc/all/dynamic-multistore/{{page.version}}/base-shop/dynamic-multistore-feature-overview.html).
```

File: docs/pbc/all/dynamic-multistore/202307.0/base-shop/dynamic-multistore-feature-overview.md [hunk 7]
```diff
@@ -1,22 +1,89 @@
...
+description: Dynamic Multistore lets you create and manage multiple online stores from the Back Office.
 last_updated: July 31, 2023
 template: concept-topic-template
 ---
 
-{% info_block warningBox %}
+The *Dynamic Multistore* feature lets you create and manage multiple online stores within the same region of the Back Office installation. It streamlines the setup and maintenance of distinct stores tailored to various customer segments, regions, or product categories. 
 
-Dynamic Multistore is part of an *Early Access Release*. This *Early Access* feature introduces the ability to handle the store entity in the Back Office. Business users can try creating stores without editing the `Stores.php` file and redeploying the system.
+In the following sections, you can learn more details about the capabilities of Dynamic Multistore.
```
Comment (andriitserkovnyi) [inline docs/pbc/all/dynamic-multistore/202307.0/base-shop/dynamic-multistore-feature-overview.md:10]: What value does this sentence give?

File: docs/pbc/all/dynamic-multistore/202307.0/base-shop/dynamic-multistore-feature-overview.md [hunk 8]
```diff
@@ -1,22 +1,89 @@
...
+
+## Adding currencies
+
+The feature also allows the addition of currencies for the created store, ensuring that transactions can be conducted in various currencies as needed.
+
+![adding-currencies](https://spryker.s3.eu-central-1.amazonaws.com/docs/pbc/all/dynamic-multistore/dynamic-multistore.md/adding-currencies.png)
+
+## Publishing and synchronizing
+
+When the new queue infrastructure is published, the store becomes visible on the storefront.
```
Comment (andriitserkovnyi) [inline docs/pbc/all/dynamic-multistore/202307.0/base-shop/dynamic-multistore-feature-overview.md:37]:
```suggestion
When the new queue infrastructure is published, the store becomes visible on the Storefront.
```

File: docs/pbc/all/dynamic-multistore/202307.0/base-shop/dynamic-multistore-feature-overview.md [hunk 9]
```diff
@@ -1,22 +1,89 @@
...
+
+## Assigning products and prices
```
Comment (andriitserkovnyi) [inline docs/pbc/all/dynamic-multistore/202307.0/base-shop/dynamic-multistore-feature-overview.md:39]:
```suggestion
## Assigning products and prices to stores
```

File: docs/pbc/all/dynamic-multistore/202307.0/base-shop/dynamic-multistore-feature-overview.md [hunk 10]
```diff
@@ -1,22 +1,89 @@
...
+
+You can assign products and their prices to a product using the Back Office UI or other methods of creating a new product in the system. This offers flexibility in managing the product catalog.
```
Comment (andriitserkovnyi) [inline docs/pbc/all/dynamic-multistore/202307.0/base-shop/dynamic-multistore-feature-overview.md:41]:
```suggestion
You can assign products and their prices to stores using the Back Office or other methods of creating products.
```

File: docs/pbc/all/dynamic-multistore/202307.0/base-shop/dynamic-multistore-feature-overview.md [hunk 11]
```diff
@@ -1,22 +1,89 @@
...
+
+The following image shows how you can assign a product to an already existing store.
+
+![store-relation](https://spryker.s3.eu-central-1.amazonaws.com/docs/pbc/all/dynamic-multistore/dynamic-multistore.md/store-relation.png)
+
+The following demonstrates shows how you can assign a product price to a specific store.
```
Comment (andriitserkovnyi) [inline docs/pbc/all/dynamic-multistore/202307.0/base-shop/dynamic-multistore-feature-overview.md:47]:
```suggestion
The following shows how you can assign a product price to a specific store.
```

File: docs/pbc/all/dynamic-multistore/202307.0/base-shop/dynamic-multistore-feature-overview.md [hunk 12]
```diff
@@ -1,22 +1,89 @@
...
+
+![product-prices](https://spryker.s3.eu-central-1.amazonaws.com/docs/pbc/all/dynamic-multistore/dynamic-multistore.md/product-prices.png)
```
Comment (andriitserkovnyi) [inline docs/pbc/all/dynamic-multistore/202307.0/base-shop/dynamic-multistore-feature-overview.md:49]: Is it just me or the picture doesn't show how to assign a price to a store?

File: docs/pbc/all/dynamic-multistore/202307.0/base-shop/dynamic-multistore-feature-overview.md [hunk 13]
```diff
@@ -1,22 +1,89 @@
...
+
+## Viewing the store on the storefront
```
Comment (andriitserkovnyi) [inline docs/pbc/all/dynamic-multistore/202307.0/base-shop/dynamic-multistore-feature-overview.md:51]:
```suggestion
## Viewing the store on the Storefront
```

File: docs/pbc/all/dynamic-multistore/202307.0/base-shop/dynamic-multistore-feature-overview.md [hunk 14]
```diff
@@ -1,22 +1,89 @@
...
+
+The newly created store is viewable on the storefront, complete with the assigned products and prices.
+
+The following example demonstrates the created store from the Back Office being in the store switcher.
```
Comment (andriitserkovnyi) [inline docs/pbc/all/dynamic-multistore/202307.0/base-shop/dynamic-multistore-feature-overview.md:55]:
```suggestion
The following example shows the store created in the Back Office being available in the store switcher on the Storefront.
```

Comment (andriitserkovnyi) [inline docs/pbc/all/dynamic-multistore/202307.0/base-shop/dynamic-multistore-feature-overview.md:55]: Once the store is published on the Storefront, customers can select it in the store switcher.

File: docs/pbc/all/dynamic-multistore/202307.0/base-shop/dynamic-multistore-feature-overview.md [hunk 15]
```diff
@@ -1,22 +1,89 @@
...
+
+![store-switcher](https://spryker.s3.eu-central-1.amazonaws.com/docs/pbc/all/dynamic-multistore/dynamic-multistore.md/store-switcher.png)
+
+The following images show that the assigned products to the newly created store are available on the storefront.
```
Comment (andriitserkovnyi) [inline docs/pbc/all/dynamic-multistore/202307.0/base-shop/dynamic-multistore-feature-overview.md:59]:
```suggestion
The product that was assigned to the store in [Assigning products and prices to stores](#assigning-products-and-prices-to-stores) is available in this store.
```

File: docs/pbc/all/dynamic-multistore/202307.0/base-shop/dynamic-multistore-feature-overview.md [hunk 16]
```diff
@@ -1,22 +1,89 @@
...
+
+![search-suggestions](https://spryker.s3.eu-central-1.amazonaws.com/docs/pbc/all/dynamic-multistore/dynamic-multistore.md/search-suggestions.png)
+
+![assigned-product-searchable-in-the-storefront](https://spryker.s3.eu-central-1.amazonaws.com/docs/pbc/all/dynamic-multistore/dynamic-multistore.md/assigned-product-searchable-in-the-storefront.png)
+ 
+## Assigning CMS Pages
+
+New stores can have CMS pages assigned, providing rich content experiences tailored to specific store needs.
+
+The following image shows the ability to assign a store to a specific CMS page to be shown on the storefront.
```
Comment (andriitserkovnyi) [inline docs/pbc/all/dynamic-multistore/202307.0/base-shop/dynamic-multistore-feature-overview.md:69]:
```suggestion
CMS pages are assigned to stores in the Back Office.
```

File: docs/pbc/all/dynamic-multistore/202307.0/base-shop/dynamic-multistore-feature-overview.md [hunk 17]
```diff
@@ -1,22 +1,89 @@
...
+
+![assigning-a-cms-page-to-the-store](https://spryker.s3.eu-central-1.amazonaws.com/docs/pbc/all/dynamic-multistore/dynamic-multistore.md/assigning-a-cms-page-to-the-store.png)
+
+The assigned CMS page is shown on the selected "SECOND" store that is selected on the Storefront.
```
Comment (andriitserkovnyi) [inline docs/pbc/all/dynamic-multistore/202307.0/base-shop/dynamic-multistore-feature-overview.md:73]:
```suggestion
When such a CMS page is published, it's only visible in the store it's assigned to.
```

File: _includes/pbc/all/install-features/202307.0/install-dynamic-multistore-the-cms-feature.md [hunk 18]
```diff
@@ -4,7 +4,7 @@ Dynamic Multistore is part of an *Early Access Release*. This *Early Access* fea
 
 {% endinfo_block %}
 
-To install Dynamic Store + the CMS feature, install the required features:
+To install the [Dynamic Multistore](/docs/pbc/all/dynamic-multistore/{{page.version}}/base-shop/dynamic-multistore-feature-overview.html) + [CMS](/docs/pbc/all/content-management-system/{{page.version}}/base-shop/cms-feature-overview/cms-feature-overview.html) feature, install the required features:
```
Comment (VadymSachenko) [inline _includes/pbc/all/install-features/202307.0/install-dynamic-multistore-the-cms-feature.md:7]:
```suggestion
To install [Dynamic Multistore](/docs/pbc/all/dynamic-multistore/{{page.version}}/base-shop/dynamic-multistore-feature-overview.html) + the [CMS](/docs/pbc/all/content-management-system/{{page.version}}/base-shop/cms-feature-overview/cms-feature-overview.html) feature, install the required features:
```

File: _includes/pbc/all/install-features/202307.0/install-dynamic-multistore-the-customer-account-management-feature.md [hunk 19]
```diff
@@ -4,7 +4,7 @@ Dynamic Multistore is part of an *Early Access Release*. This *Early Access* fea
 
 {% endinfo_block %}
 
-This document describes how to install Dynamic Store + the Customer Account Management feature.
+This document describes how to install the [Dynamic Multistore](/docs/pbc/all/dynamic-multistore/{{page.version}}/base-shop/dynamic-multistore-feature-overview.html) + [Customer Account Management](/docs/pbc/all/customer-relationship-management/{{page.version}}/base-shop/customer-account-management-feature-overview/customer-account-management-feature-overview.html) feature.
```
Comment (VadymSachenko) [inline _includes/pbc/all/install-features/202307.0/install-dynamic-multistore-the-customer-account-management-feature.md:7]:
```suggestion
This document describes how to install [Dynamic Multistore](/docs/pbc/all/dynamic-multistore/{{page.version}}/base-shop/dynamic-multistore-feature-overview.html) + the [Customer Account Management](/docs/pbc/all/customer-relationship-management/{{page.version}}/base-shop/customer-account-management-feature-overview/customer-account-management-feature-overview.html) feature.
```

File: _includes/pbc/all/install-features/202307.0/install-dynamic-multistore-the-prices-feature.md [hunk 20]
```diff
@@ -4,7 +4,7 @@ Dynamic Multistore is part of an *Early Access Release*. This *Early Access* fea
 
 {% endinfo_block %}
 
-This document describes how to install Dynamic Store + the Prices feature.
+This document describes how to install the [Dynamic Multistore](/docs/pbc/all/dynamic-multistore/{{page.version}}/base-shop/dynamic-multistore-feature-overview.html) + [Prices](/docs/pbc/all/price-management/{{page.version}}/base-shop/prices-feature-overview/prices-feature-overview.html) feature.
```
Comment (VadymSachenko) [inline _includes/pbc/all/install-features/202307.0/install-dynamic-multistore-the-prices-feature.md:7]:
```suggestion
This document describes how to install [Dynamic Multistore](/docs/pbc/all/dynamic-multistore/{{page.version}}/base-shop/dynamic-multistore-feature-overview.html) + the [Prices](/docs/pbc/all/price-management/{{page.version}}/base-shop/prices-feature-overview/prices-feature-overview.html) feature.
```

File: _includes/pbc/all/install-features/202307.0/install-dynamic-multistore-the-product-feature.md [hunk 21]
```diff
@@ -4,7 +4,7 @@ Dynamic Multistore is part of an *Early Access Release*. This *Early Access* fea
 
 {% endinfo_block %}
 
-This document describes how to install Dynamic Store + the Product feature.
+This document describes how to install the [Dynamic Multistore](/docs/pbc/all/dynamic-multistore/{{page.version}}/base-shop/dynamic-multistore-feature-overview.html) + [Product](/docs/pbc/all/product-information-management/{{page.version}}/base-shop/feature-overviews/product-feature-overview/product-feature-overview.html) feature.
```
Comment (VadymSachenko) [inline _includes/pbc/all/install-features/202307.0/install-dynamic-multistore-the-product-feature.md:7]:
```suggestion
This document describes how to install [Dynamic Multistore](/docs/pbc/all/dynamic-multistore/{{page.version}}/base-shop/dynamic-multistore-feature-overview.html) + the [Product](/docs/pbc/all/product-information-management/{{page.version}}/base-shop/feature-overviews/product-feature-overview/product-feature-overview.html) feature.
```

File: docs/pbc/all/dynamic-multistore/202307.0/base-shop/dynamic-multistore-feature-overview.md (hunk 7 above)
Comment (VadymSachenko) [reply docs/pbc/all/dynamic-multistore/202307.0/base-shop/dynamic-multistore-feature-overview.md:10]: It just seemed right to add an intro sentence for the sections

File: docs/pbc/all/dynamic-multistore/202307.0/base-shop/dynamic-multistore-feature-overview.md (hunk 12 above)
Comment (VadymSachenko) [reply docs/pbc/all/dynamic-multistore/202307.0/base-shop/dynamic-multistore-feature-overview.md:49]: @andriitserkovnyi Not sure what you want me or @a-sabaa to do about it.

File: docs/pbc/all/dynamic-multistore/202307.0/base-shop/delete-stores.md [hunk 22]
```diff
@@ -11,49 +11,58 @@ Dynamic Multistore is currently running under an *Early Access Release*. Early A
...
+
+{% info_block infoBox %}
+
+The database data to modify in this section is relevant to the default configuration. Depending on the features your project has, you may need to modify more data.
+
+{% endinfo_block %}
 
-The number of tables that use relation with the `spy_store` table may depend on the use of the functionality, and on the installed modules.
 
 The list of available tables with data that may contain a foreign key relationship:
```
Comment (andriitserkovnyi) [inline docs/pbc/all/dynamic-multistore/202307.0/base-shop/delete-stores.md:67]: What should I do with it?

File: docs/pbc/all/dynamic-multistore/202307.0/base-shop/delete-stores.md [hunk 23]
```diff
@@ -86,94 +95,94 @@ The list of available tables with data that may contain a foreign key relationsh
 - `spy_touch_search`
 - `spy_touch_storage`
 
-Store has new configuration tables that were used for dynamic store:
+Store has new configuration tables that are used for dynamic store:
```
Comment (andriitserkovnyi) [inline docs/pbc/all/dynamic-multistore/202307.0/base-shop/delete-stores.md:98]: What should I do with these tables?

File: docs/pbc/all/dynamic-multistore/202307.0/base-shop/delete-stores.md [hunk 24]
```diff
@@ -86,94 +95,94 @@ The list of available tables with data that may contain a foreign key relationsh
...
+With Elasticsearch, each configured store has its index, which is installed automatically. An indexed name consists of the following parts:
+- An optional prefix, which is defined by the `SearchElasticsearchConstants::INDEX_PREFIX` configuration option. For example, we use `spryker` by default.
 - A store name.
 - A configuration file name.
-Index name components are delimited with an underscore, for example:  `spryker_xxx_page`.
 
-The following indexes are available in the standard configuration (example: `xxx` -  store name):
+Index name components are delimited with an underscore—for example, `spryker_xxx_page`.
+
+The following indexes are available in the standard configuration:
```
Comment (andriitserkovnyi) [inline docs/pbc/all/dynamic-multistore/202307.0/base-shop/delete-stores.md:164]: What should I do with them?

File: docs/pbc/all/dynamic-multistore/202307.0/base-shop/delete-stores.md [hunk 25]
```diff
@@ -86,94 +95,94 @@ The list of available tables with data that may contain a foreign key relationsh
...
+
+2. Adjust `kv:store_list:` and delete the store name XXX values in stores.
 ```json
 {"stores":["AT","DE","XXX"],"_timestamp":111111111111}
 ```
-- Delete key `kv:store:xxx` with value about store.
+3. Delete the `kv:store:xxx` key with the store value.
 
 
 Note: A project's custom data stored keys may also be present.
```
Comment (andriitserkovnyi) [inline docs/pbc/all/dynamic-multistore/202307.0/base-shop/delete-stores.md:153]: What are we trying to explain here?

**Pull request merged**